src/esw/auton_led.py

```diff
@@ -10,8 +10,7 @@
 )
 
 ser = serial.Serial
```

Comment:
ser does not need to be global anymore either

Reply:
done